### PR TITLE
updated intro text to match date and time pickers

### PR DESCRIPTION
**Pull request by Blackbaud-DeniseCisneros**
Addresses issue https://github.com/blackbaud/skyux2-docs/issues/297

### CODE REVIEW

File: src/app/components/colorpicker/index.html [hunk 1]
```diff
@@ -57,23 +62,19 @@
...
       propertyName="presetColors"
       defaultValue="['#333', '#888', '#EFEFEF', '#FFF']"
       isOptional="true">
-      Specifies an array of colors to load in the colorpicker as preset choices. The colorpicker displays the colors in a series of 12 boxes for users to select.
-    </sky-demo-page-property>
-    <sky-demo-page-property
-      propertyName="skyColorpickerInput">
-      Place this attribute on an input element or button element that is wrapped in a <stache-code>sky-colorpicker</stache-code> component to create the element and colorpicker. You must set this attribute to the instance of the <stache-code>sky-colorpicker</stache-code>. This directive uses <stache-code>NgModel</stache-code> to bind data.
+      Specifies an array of colors to load as preset choices. The colorpicker displays the colors in a series of 12 boxes for users to select.
     </sky-demo-page-property>
```
Comment (blackbaud-johnly) [inline src/app/components/colorpicker/index.html]: We are missing a public `returnFormat` input: https://github.com/blackbaud/skyux-colorpicker/blob/master/src/app/public/modules/colorpicker/colorpicker-input.directive.ts#L92. Looking at the PRs in the repo, it looks like Trevor and Steve are the ones who worked on this component most recently. Can you check with them to see if we should add the `returnFormat` property to the docs?

File: src/app/components/colorpicker/index.html [hunk 2]
```diff
@@ -42,11 +42,16 @@
...
-        </pre></div>
+        </pre>
+      </div>
     </sky-demo-page-property>
   </sky-demo-page-properties>
 
   <sky-demo-page-properties sectionHeading="Colorpicker input directive properties">
+    <sky-demo-page-property
+      propertyName="skyColorpickerInput">
+      Place this attribute on an input element or button element that is wrapped in a <stache-code>sky-colorpicker</stache-code> component to create the element and colorpicker. You must set this attribute to the instance of the <stache-code>sky-colorpicker</stache-code>. This directive uses <stache-code>NgModel</stache-code> to bind data.
```
Comment (blackbaud-johnly) [inline src/app/components/colorpicker/index.html]: I'd reword this to mimic the wording for `skyDatepickerInput` in datepicker and `skyTimepickerInput` in timepicker: "Creates the colorpicker element and dropdown. Place this attribute on an `input` element or `button` element, and wrap the element in a `sky-colorpicker` component. The attribute must be set to the instance of the `sky-colorpicker` component."

File: src/app/components/colorpicker/index.html (hunk 1 above)
Comment (Blackbaud-DeniseCisneros) [reply src/app/components/colorpicker/index.html]: Confirmed with Trevor that the property has never been used and doesn't need documented.
